resources/pdns_recursor_install_rhel.rb

```diff
@@ -66,3 +69,9 @@
     action :remove
   end
 end
+
+action_class.class_eval do
+  def whyrun_supported?
```

Comment:
Are you sure about this? Because there is an issue open for that: https://github.com/chef/chef/issues/4537